src/components/Banner/AndroidAnnouncementBanner/styled.tsx

```diff
@@ -0,0 +1,96 @@
+import { ButtonText } from 'components/Button'
+import { OpacityHoverState } from 'components/Common'
+import { X } from 'react-feather'
+import styled from 'styled-components'
+import { BREAKPOINTS } from 'theme'
+import { Z_INDEX } from 'theme/zIndex'
+
+export const PopupContainer = styled.div<{ show: boolean }>`
+  ${({ show }) => !show && 'display: none'};
+
+  background-color: ${({ theme }) => theme.surface2};
+  color: ${({ theme }) => theme.neutral1};
+  position: fixed;
+  z-index: ${Z_INDEX.sticky};
+
+  border-radius: 20px;
+  bottom: 40px;
+  right: 20px;
+  width: 360px;
+  height: 92px;
+
+  border: 1.3px solid ${({ theme }) => theme.surface3};
+
+  @media only screen and (max-width: ${BREAKPOINTS.md}px) {
+    bottom: 62px;
+  }
+
+  @media only screen and (max-width: ${BREAKPOINTS.xs}px) {
+    width: unset;
+    right: 10px;
+    left: 10px;
+  }
+
+  user-select: none;
```

Comment:
I would move the `user-select` rule up and remove the whitespace amongst these rules.